react/week1/todo-list-app/src/App.js

```diff
@@ -0,0 +1,23 @@
+import logo from './logo.svg';
+import './App.css';
+import Todolist from './Components/Todolist';
+import Getup from './Components/Getup'
+import Brush from './Components/Brush';
+import Eat from './Components/Eat';
+import Useclasses from './Components/Useclasses'
+
+function App() {
+  return (
+    <div className="App">
+      <Todolist />
+      {/* <Getup description = "Get out of bed" deadline = "Wed Sep 13 2017" /> */}
```

Comment:
we need to create components which will take data from parent to child and show data, here you are calling the same component multiple times, can you make it in loop and provide array of data. Remember whats Niels said in class its all about separation

Reply:
Thanks for your feedback Piyush, but unfortunately I'm lost, I tried to use the map method to do it in a loop but I failed. so I donno how to this :(